### DISCUSSION TITLE
Release 0.15 [FUNCTIONAL]

### DISCUSSION THREAD

**Discussion by nmsderp** [cytrinc/bark-coding#63] [General]
a actual functional release

  

  

 
 

  
  
 

 
 

it is already copy and pasted into nw.js,

<hr /><em>This discussion was created from the release <a href='https://github.com/mariocraft987/bark.github.io/releases/tag/'>Release 0.15 [FUNCTIONAL]</a>.</em>

**Comment by rubiidev18alt** [comment]
uh, maybe someone can create a desktop version that already is copy and pasted into nw.js

**Comment by nmsderp** [comment]
> uh, maybe someone can create a desktop version that already is copy and pasted into nw.js
what do you mean, it is copy and pasted, that was just me telling you how
